app/main.py

```diff
@@ -2,24 +2,91 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: object, owner: object) -> int:
+        return getattr(instance, self._name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise ValueError("Value must be an integer")
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError(f"Value must be between {self.min_amount}"
+                             f" and {self.max_amount}")
+        setattr(instance, self._name, value)
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self._name = name
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(
+            self,
+            age: IntegerRange(int, int),
+            weight: IntegerRange(int, int),
+            height: IntegerRange(int, int)
+    ) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    def __init__(self) -> None:
+        super().__init__(
+            IntegerRange(4, 14),
+            IntegerRange(20, 50),
+            IntegerRange(80, 120)
+        )
```

Comment:
```suggestion
        age = IntegerRange(4, 14)
        weight = IntegerRange(20, 50)
        height  = IntegerRange(80, 120)
```
Set them as descriptors